src/utils/star-canvas.ts

```diff
@@ -29,7 +30,7 @@ let lineLength = scale * 100; // Length of each line
 const starColor = "#fffceb";
 const centerX = Math.round(canvas.width / 2);
 const centerY = Math.round(canvas.height / 2);
-const pointSize = scale * 3; // Size of each square point
+const pointSize = scale * (dpr === 1 ? 3 : 3.25); // Size of each square point
```

Comment:
The star looked weird with the existing numbers after scaling, on both screen sizes but especially mobile. I don't really understand why exactly, probably some intricate ratio dance whose ideal number changes slightly with the slight dpi scaling. So this line scales it up on high-dpi devices but keeps it the same on lower-dpi devices.